### PR TITLE
Update Koreapi to use a proper library for CSV serialization. 

### PR DESCRIPTION
**Pull request by housepage**
Add .csv extension to download file. Clean up of serialization code.

### CODE REVIEW

**Comment by housepage** [comment]
@mhotan Give this a look when you come in before Monday morning? Thanks!

File: koreapi.default.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
-environment = development
-core.server.host = 127.0.0.1
+environment = production
+core.server.host = eol03
 core.server.port = 8081
-metric-config.balboa.server.host = 127.0.0.1
+metric-config.balboa.server.host = eol03
```
Comment (mhotan) [inline koreapi.default.properties]: We should probable make this repo private due to the content.  Agreed?

File: scriptlets/domain_report.js [hunk 2]
```diff
@@ -28,38 +34,49 @@ runAndWriteToFile = function () {
               scriptlet.log("Already processed domain " + domainName);
               continue
           }
-          uniqueDomainIds[domainId] = "";
+          uniqueDomainIds[domainId] = "Value That Does Not Matter";
```
Comment (mhotan) [inline scriptlets/domain_report.js]: Is this default assignment to a literal String value a javascript thing I do not understand?

File: koreapi.default.properties (hunk 1 above)
Comment (sokrahta) [inline koreapi.default.properties]: or make the hostname a config at deploy time

**Comment by mhotan** [comment]
Otehr then javascript things I do not understand or like this is a reasonable change that gives us some great short term wins.  I feel the domain export functionality of koreapi
 should really be moved into a chronos job written in Scala at some point.  Thank you for doing this @housepage.  :two_men_holding_hands:

File: scriptlets/domain_report.js (hunk 2 above)
Comment (housepage) [inline scriptlets/domain_report.js]: They're using maps to implement sets in Javascript.

**Pull request merged**